understanding/20/audio-only-live.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Audio-only (Live)</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Live audio can be understood by more people</dd>
-         <dt>Author task</dt><dd>Provide a text equivalent for live audio-only content</dd>
+         <dt>Goal</dt><dd>Live audio can be understood by more people.</dd>
+         <dt>What to do</dt><dd>Provide a text equivalent for live audio-only content.</dd>
+         <dt>Why it's important</dt><dd>People who are deaf or hard of hearing can understand real-time audio content.</dd> 
```

Comment:
@rainbreaw commented:
> Similarly, in understanding/20/audio-only-live.html, understanding/20/captions-live.html and understanding/20/captions-prerecorded.html:

> Consider extending why it's important not just to include individuals who are deaf or hard of hearing, but also to include "and individuals with a cognitive disability that impacts attention and working memory, so that they have a way to access what they may have missed."

Reply:
I don't think this applies to live captions, since they are real time.

Reply:
@GreggVan replied via email
> I had the same thought — but then thought of live text transcripts which are provided for some meetings.   And with new AI recognition — it is available for zoom and skype calls.  So this does apply to live content now.  Or can now.   And that is the purpose of this SC   no?

The Understanding document talks about live transcripts as a potential _source_ for the live stream (instead of someone transcribing it in real time, which is preferred). 
Any record of the live session would logically become 'pre-recorded' and follow the requirements of a different SC. IMO opinion, the "live" requirements are literally the expectations of what to offer in real time. As such, considerations of them being an aid for "what they may have missed" cannot apply.
Note that the lines are being blurred somewhat with newer technologies, but I believe the reason for distinuishing between a recording and live persists and makes sense to divide author responsibilities

Reply:
@mbgower this does apply to real-time captions as well as live transcription. Evidence shows that some individuals can benefit from the combination of the captions and the audio at the same time to support attention, comprehension, and retention of what is being said.

Without spending a lot of time identifying academic evidence, here is one research synthesis that documents some of these benefits: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5214590/

Reply:
ah, you didn't mean "what they missed" in terms of scrolling back and looking at prior comments, but of reinforcing visually what was spoken! Okay, interesting...

I note that the Understanding document lacks a Benefits section (!) but does say 
> A live text caption service will enable live audio to be accessible to people who are deaf or hard of hearing, or who cannot otherwise hear the audio.

I think we've got our phrase from that, and am going to suggest
```suggestion
         <dt>Why it's important</dt><dd>People who cannot hear or understand real-time audio can read the content.</dd> 
```